packages/web-components/fast-foundation/package.json

```diff
@@ -96,7 +96,7 @@
   },
   "dependencies": {
     "@floating-ui/dom": "^1.0.3",
-    "@microsoft/fast-element": "^2.0.0-beta.22",
+    "@microsoft/fast-element": "2.0.0-beta.22",
```

Comment:
I think some investigation is needed on the impact this will have on the beachball releases and vice versa. I'm not sure if beachball just increments the version number, or if it replaces the entire line. I would hate to see this change get reversed by beachball on the next publish.

Reply:
Maybe @chrisdholt can comment on the beachball specifics, but I think you can run `beachball bump` locally to see what it does to the dependent packages.

Reply:
Agreed. Do you have any instructions on how we currently make a release? I could test it out. Is it using `generateChangelog` from https://microsoft.github.io/beachball/overview/configuration.html#options

Reply:
I can pull this down and attempt to check, but I don't actually think this is going to work. Even if we were to fix the versions for a moment, I think that any update like this is likely to be overwritten when we're bumping. I can pull down your branch and try.

What _could_ possibly be done - though I don't think we will know until it's been done would be to fix the version **at publish time**. Currently we bump and release with beachball and then publish manually. So, technically, we could (so long as we remember) try and fix the fast-element version to fast-foundation.

I do want to note here though - I don't think that we would maintain this in the future once we're out of alpha/beta.

Reply:
Appreciate that, Chris. Any solution until we are out of alpha/beta would be highly desirable. We started using the alpha/beta and they have been pretty stable. I would hate to have our many teams be blocked by this. We are heavily building against Beta and Alpha versions.

Reply:
Hey @chrisdholt . Hope all is well. Wonder if you got a chance to test? I think if we can have an alpha version that has a fixed version of fast-element would be great in the next release. That way, we will have a "stable" dependency on FAST alpha and beta until there is a better solution.

Reply:
I have another monorepo with the dependency version locked and just ran `beachball bump` and it looks like it didn't change the dependencies to "^VERSION"

Reply:
The problem here is that I think we only want to fix these _for publishes_. We shouldn't rev this via PR every time we need to publish and then move it back for local development. Curious of @nicholasrice's thoughts here, but I'd prefer we just manually do this. If we have things that have completed in FAST Element or Foundation I'd be happy to try and bump with this manual process tomorrow...But I don't think we want to be developing and merging PR's back and forth.

Reply:
I'm not following the concern with this. My expectation is that after removing the caret version selector, beachball will increment the version the same way it does today during our manual publish process, but the specific version will be targeted instead of the caret version.

When it comes time to come out of alpha/beta, we'll re-add the caret version selector because we won't add breaking changes w/o major version iteration.

Reply:
Sure, perhaps my concern isn't warranted - my immediate thought is that if we lock the version of FE to the the published version of the package, will local development still symlink with what is in the repo or will it resolve to the published version? If it resolves to the publish version, we're developing Foundation a version back and not actually with the local, symlinked version, which could cause sync issues when changes come into FE...